Traceback (most recent call last):
  File "C:\Anaconda3\envs\akfds-dev\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\king\AppData\Roaming\Python\Python38\site-packages\nbclient\client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\king\AppData\Roaming\Python\Python38\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\king\AppData\Roaming\Python\Python38\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\Anaconda3\envs\akfds-dev\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "C:\Users\king\AppData\Roaming\Python\Python38\site-packages\nbclient\client.py", line 535, in async_execute
    await self.async_execute_cell(
  File "C:\Users\king\AppData\Roaming\Python\Python38\site-packages\nbclient\client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\king\AppData\Roaming\Python\Python38\site-packages\nbclient\client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import akshare as ak
ak.__version__

stock_zh_a_hist_df = ak.stock_zh_a_hist(symbol="000001")
print(stock_zh_a_hist_df)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [1;32mIn [12][0m, in [0;36m<cell line: 1>[1;34m()[0m
[1;32m----> 1[0m [38;5;28;01mimport[39;00m [38;5;21;01makshare[39;00m [38;5;28;01mas[39;00m [38;5;21;01mak[39;00m
[0;32m      2[0m ak[38;5;241m.[39m__version__
[0;32m      4[0m stock_zh_a_hist_df [38;5;241m=[39m ak[38;5;241m.[39mstock_zh_a_hist(symbol[38;5;241m=[39m[38;5;124m"[39m[38;5;124m000001[39m[38;5;124m"[39m)

[1;31mModuleNotFoundError[0m: No module named 'akshare'
ModuleNotFoundError: No module named 'akshare'

